src/components/GlobalPreferences/Notifications/NotificationsPreferences.js

```diff
@@ -0,0 +1,170 @@
+import React, { useCallback } from 'react'
+import {
+  Box,
+  Button,
+  GU,
+  IconMail,
+  IconTrash,
+  Info,
+  Link,
+  Switch,
+  useTheme,
+  Split,
+  Tag,
+  textStyle,
+} from '@aragon/ui'
+
+import emailNotifcationIllustration from '../../../../src/assets/emailNotifications.svg'
+
+export default function NotificationsPreferences({
+  email,
+  notificationsDisabled,
+  onSwitchNotificationsStatus,
+  onLockSettings,
+  onDeleteEmail,
+  onUpdateEmail,
+}) {
+  const theme = useTheme()
+
+  return (
+    <React.Fragment>
```

Comment:
Nitpick: should we consolidate usage of `<React.Fragment>` and `<>`? I think there are a few other PRs just using `<>`.